rootfs/etc/nginx/lua/util.lua

```diff
@@ -146,6 +146,11 @@ function _M.is_blank(str)
   return str == nil or string_len(str) == 0
 end
 
+function _M.string_to_bool(str)
+  if str == "true" then return true end
+  return false
```

Comment:
I'm not much into Lua, but `return str == "true"` doesn't work here?

Reply:
yes, good catch. fixed here